spec/features/facilities/index_spec.rb

```diff
@@ -29,7 +29,11 @@
   end
 
   it "should allow exporting to csv" do
-    facilities = create_list(:facility, 5, organization: user.organization)
+    facility_codes = %w[vortex anemo dendro pyro electro]
+    facilities = []
+    facility_codes.each do |code|
+      facilities << create(:facility, code: code, organization: user.organization)
+    end
```

Comment:
Instead of this, could we update the [facility factory](https://github.com/rubyforgood/abalone/blob/master/spec/factories/facility.rb) to use a [sequence](https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#sequences)? The `create_list` approach should still work in that case.

Reply:
Yes, thanks for taking on this issue @nsiregar. I agree, if we could use the `create_list` approach here, that would be great.

Reply:
Sure, we can use factory sequence